soh/soh/Enhancements/debugger/performanceTimer.cpp

```diff
@@ -1,5 +1,3 @@
-#pragma once
-
```

Comment:
This pragma should be turned into the `#ifndef _H #define _H #endif` pattern so that declarations aren't repeated. This is the formal way of doing what `pragma once` was doing.

Reply:
This is in a cpp file, not a header file. I checked that we don't have any code including cpp files

Reply:
Ah, I'm blind. Carry on.